_includes/api/en/4x/req-fresh.md

```diff
@@ -2,12 +2,19 @@
 
 When the response is still "fresh" in the client's cache `true` is returned, otherwise `false` is returned to indicate that the client cache is now stale and the full response should be sent.
 
+In order to check whether the response is still "fresh" in the client's cache or not, two tags will be used ETag and If-None-Match. ETag will be fetched from response (saved internally from application) and other tag will be received from client's request.
+
 When a client sends the `Cache-Control: no-cache` request header to indicate an end-to-end reload request, this module will return `false` to make handling these requests transparent.
 
 Further details for how cache validation works can be found in the
 [HTTP/1.1 Caching Specification](https://tools.ietf.org/html/rfc7234).
 
 ```js
+
+// Get value for ETag where you saved it initially, In general you 
```

Comment:
This comment isn't very clear. The added example is setting an etag not reading a value.
users do not need to manage or store timestamps to get etags working, so I think this will confuse people